command/commands.go

```diff
@@ -517,6 +517,21 @@ func Commands(metaPtr *Meta, agentUi cli.Ui) map[string]cli.CommandFactory {
 				Meta: meta,
 			}, nil
 		},
+		"operator raft _info": func() (cli.Command, error) {
```

Comment:
This may be nitpicky but I'm wondering if we should make it even more obvious that these commands only run locally? `operator` commands I would think would correspond to a similar `/v1/operator` endpoint. Since they are internal and `_` prefixed it may not matter too much.

Maybe `nomad debug local raft _info` or `nomad local raft _info`? Or operator is fine too, just wanted to see what you though

Reply:
Since they're missing from help and prefixed, I think that's good enough. I think we should also note there are no guarantees with these commands (other than that they are *read only*).